src/components/staking/LiValidator.vue

```diff
@@ -49,9 +49,14 @@
       </div>
     </td>
     <td :class="{ 'hide-xs': showOnMobile !== 'expectedReturns' }">
-      {{
-        validator.expectedReturns ? percent(validator.expectedReturns) : `--`
-      }}
+      <span v-if="network === `emoney-testnet`">
```

Comment:
nooooooop, no network specific fixes please. why not apply the fix for all?

Reply:
I just want to hear from you :-)

Reply:
i may have mislead mario here 🙃